src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -6,6 +6,13 @@
 
 import createLinkFromAudioBuffer from './exporter.mjs';
 
+// This enum states the current recording state
+const recorderState = {
```

Comment:
https://google.github.io/styleguide/jsguide.html#jsdoc-enum-and-typedef-comments

For enum, we want to capitalize the first letter. e.g. `RecorderState`